prereise/gather/demanddata/nrel_efs/get_efs_data.py

```diff
@@ -36,129 +33,273 @@ def download_data(es=None, ta=None, fpath=""):
     if ta is None:
         ta = {"All"}
 
-    # Check that the inputs are of an appropriate type
+    # Check the inputs
+    es = _check_electrification_scenarios_for_download(es)
+    ta = _check_technology_advancements_for_download(ta)
+    fpath = _check_path(fpath)
+
+    # Download each of the specified load profiles
+    z = {}
+    for i in es:
+        z[i] = {}
+        for j in ta:
+            # Assign path and file names
+            zip_name = "EFSLoadProfile_" + i + "_" + j + ".zip"
```

Comment:
We are only testing against Python 3.7+, so we can make use of f-strings to simplify the construction of these filenames (here and elsewhere). E.g.: `"EFSLoadProfile_" + i + "_" + j + ".zip"` -> `f"EFSLoadProfile_{i}_{j}.zip"`.

Reply:
Done.